src/app/home-page/widget/news-updates/news-updates.component.scss

```diff
@@ -2,3 +2,8 @@
   padding: 1rem;
   border-bottom: 0.1rem solid #e0e1e2;
 }
+
+.subtitle {
```

Comment:
Duplicate of the one in src/app/home-page/recent-events/recent-events.component.scss  .  Move them to styles.scss, might have to rename the class to something more specific just in case it conflicts with other components on site.

Reply:
Since they are in different usage contexts, I probably should just give it a unique name here so it can be independent of the `recent-events` one.

Reply:
No, we do that way too often and then end up with conflicting styles and then the site looks inconsistent.  Since the two classes are exactly identical anyways, have them reference the same class.